src/fabric/src/fabric_ring.erl

```diff
@@ -133,6 +140,10 @@ handle_response(Shard, Response, Workers, Responses, RingOpts, CleanupCb) ->
             #shard{range = [B, E]} = Shard,
             Responses1 = [{{B, E}, Shard, Response} | Responses],
             handle_response_ring(Workers1, Responses1, CleanupCb);
+        [{rmin, RMin}] when is_integer(RMin), RMin >= 1 ->
```

Comment:
Since this is an internal function, the assumption is that by now the value of R has been checked above somewhere and this is more of an assert. It also goes along with the pattern in this function where we'd blow up with a case clause on unexpected `RingOpts` values.

In general we could conceive of a case where R=0 means returns even if we we haven't completed the ring even once but it's more likely that we've just forgot to validate its value and that's not something we meant to do.